bin/tinyreq

```diff
@@ -52,5 +56,5 @@ tinyreq(options, function (err, body) {
     if (err) {
         return logger.log(err, "error");
     }
-    console.log(body);
+    logger.log(body);
```

Comment:
Keep the `console.log` because `logger.log` adds the type, which we don't want in case of body.